functional/tdx/run.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+script_path=$(dirname "$0")
+source "${script_path}/../../lib/common.bash"
+
+runtime_type="io.containerd.kata.v2"
+config_file=""
+kernel_tdx_path="/usr/share/kata-containers/vmlinuz-tdx.container"
+qemu_tdx_path="/usr/local/bin/qemu-system-x86_64"
+container_name=test
+FIRMWARE="${FIRMWARE:-}"
+FIRMWARE_VOLUME="${FIRMWARE_VOLUME:-}"
+
+trap cleanup EXIT
+
+get_config_file() {
+	if [ -z "${config_file}" ]; then
+		for f in $(kata-runtime --show-default-config-paths); do
+			[ -f "${f}" ] && config_file="${f}" && break
+		done
+	fi
+	echo "${config_file}"
+}
+
+cleanup() {
+	local config_file="$(get_config_file)"
+	sudo mv -f "${config_file}.bak" "${config_file}"
+
+	sudo ctr t kill -s 15 ${container_name} || true
+	sudo ctr t rm -f ${container_name} || true
+	sudo ctr c rm ${container_name} || true
+
+	clean_env_ctr
+}
+
+enable_confidential_computing() {
+	local conf_file="$(get_config_file)"
+	[ -n "${conf_file}" ] || die "configuration file not found"
+	sudo sed -i -e 's|path =.*|path = "'${qemu_tdx_path}'"|' \
```

Comment:
`sed` is awesome but `crudini` might make these changes clearer.

Reply:
And I've been using `sed` all that time... :(